src/components/chat-item/chat-item-information-card.ts

```diff
@@ -0,0 +1,109 @@
+import { DomBuilder, ExtendedHTMLElement } from '../../helper/dom';
+import { ChatItemContent, ChatItemType, Status } from '../../static';
+import { Icon, MynahIcons } from '../icon';
+import { ChatItemCard } from './chat-item-card';
+
+export interface ChatItemInformationCardProps {
+  tabId: string;
+  messageId: string | undefined;
+  informationCard: {
```

Comment:
why not using and extending the `ChatItemContent` type?
Also, having the `testId` property might be useful here as well.